device_manufacturer_partnerships/device_manufacturer_partnerships.model.lkml

```diff
@@ -0,0 +1,2 @@
+connection: "telemetry"
+label: "Device Manufacturer Partnerships"
```

Comment:
I believe here in the model you also need to have an include to your explore:
https://github.com/mozilla/looker-spoke-default/blob/767049922ec1027419f42586585dca0c1e5dea13/fenix/fenix.model.lkml#L4